rllib/utils/typing.py

```diff
@@ -143,5 +144,26 @@
 # (possibly nested) dict|tuple of gym.space.Spaces.
 SpaceStruct = Union[gym.spaces.Space, dict, tuple]
 
+# A batch of RNN states with dimensions [state_index, batch, state_object].
+StateBatch = List[List[Any]]
+
+# Format of data output from policy forward pass.
+PolicyOutputType = Tuple[TensorStructType, StateBatch, Dict]
```

Comment:
Nice, so this is basically the return structure for `compute_actions()` then.
Good thing to standardize this via this type and typehint it everywhere.

Reply:
yep, that's the idea.